EN4_processing/GEN_MOD_Dave_example_profile_validation.py

```diff
@@ -168,6 +168,8 @@
 
 # Extract time indices between start and end dates for Profile data.
 t_ind = pd.to_datetime(profile.dataset.time.values)>=start_date
+
+profile.dataset = profile.dataset.rename({"profile": "id_dim"})
```

Comment:
@endaodea I am a bit surprise you had to add this line. Any idea why the try/except statement on line 158 (pasted below for convenience) didn't catch this problem and do the `.rename({"profile": "id_dim"})` then?

line 158
```
try:
  profile.dataset = xr.open_dataset(fn_prof, chunks={'id_dim':10000})
except:
  profile.dataset = xr.open_dataset(fn_prof, chunks={'profile':10000})
  profile.dataset = profile.dataset.rename({"profile":"id_dim"})
  print(f"Profiles generated with legacy COAsT version. New dims: (id_dim, z_dim)")
```

Reply:
hmm its possible I added that before the afternoon ill take it out and test it again.

Reply:
@jpolton OK reproduced old coast files and ran without the extra line and it fails complaining about expected id_dim, so the try line must not quite be working as expected but works with the extra rename

Reply:
@jpolton maybe for chunking, it does not care if it can or cannot find the dimension in question?

Reply:
@endaodea OK. I think the try/except needs a "test if id_dim dimension exists" instead. I will have a think about the syntax unless you get there first.

Reply:
@endaodea I've pushed an edit to expand the try/except statements so that bundles the loading of data and the time slicing together. Though repetitive, when legacy COAsT issues are history, this can easily be trimmed by removing the except portion.

Reply:
@jpolton 
Looks good, Tried it on sample data with profile dimension and works as expected.